sdk/python/cmd/pulumi-language-python-exec

```diff
@@ -152,6 +152,8 @@ if __name__ == "__main__":
         exit_code = 0
     except pulumi.RunError as e:
         pulumi.log.error(str(e))
+    except ImportError as e:
+        pulumi.log.error(str(e))
```

Comment:
This will also swallow errors if a user tries to import a module that doesn't exist in their program and hide stack traces.

I'm trying to think through ways to selectively hide things, but they're all kinda hacky.